### DISCUSSION TITLE
显示显卡错误的弹窗后，重装显卡驱动后错误弹窗依然出现

### DISCUSSION THREAD

**Discussion by P-C-LION** [Meloong-Git/PCL#2671] [提问] [answered]
显示显卡错误的弹窗后，重装显卡驱动后错误弹窗依然出现，显卡为AMD Radeon Graphics

**Comment by WForst-Breeze** [comment, marked as answer]
请提供相关日志

**Comment by P-C-LION** [comment]
这是文件





---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年9月7日(周四) 晚上6:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hex-Dragon/PCL2] 显示显卡错误的弹窗后，重装显卡驱动后错误弹窗依然出现 (Discussion #2671)




 
请提供相关日志

**Reply by WForst-Breeze** [reply]
不要用邮件回复（）我们看不到邮件里附带的附件，在 Github 上将你的附件拖入输入框，等待上传完成后发送回复
